### ISSUE TITLE
Unstyled button styles

### ISSUE THREAD

**Issue by latentflip** [andyet/yeti.css#69]
One thing I've learned recently, is that we should be using buttons more often than we are doing in apps where the thing is an action - particularly important for screenreaders etc. Often though we want buttons to _look_ the same as links, take this dropdown from requiresafe:

![requiresafe portal from lift security 2015-02-04 10-34-39](https://cloud.githubusercontent.com/assets/78225/6038921/6f205e6c-ac59-11e4-8a65-3560f4f74265.png)

It'd be great if we had a `.button-unstyled` class or something, that looked like a link. And also, wherever we are currently styling links, like in dropdowns, if those buttons could have the same styling.

I've made this work for that specific example like this:

```
//style .button-unstyled like a link
.button-unstyled
  @extend a
  background: none
  border: none

//make unstyled buttons in dropdowns like links
.dropdown .button-unstyled
  padding: 10px
```

but it'd be great if this worked across all of yeticss. What do you think?
